{
    "output": "Traceback (most recent call last):\n  File \"/home/matt/Dropbox/teach/gradescope_auto_py/test/autograder/prep.py\", line 8, in <module>\n    some_list_too_short[10]\nIndexError: list index out of range\n",
    "tests": [
        {
            "max_score": 1.0,
            "name": "assert get_circle_area(radius=1) == pi, 'get_circle_area(radius=1)  (1 pts)'",
            "score": 1.0,
            "visibility": "visible"
        },
        {
            "max_score": 3.0,
            "name": "assert get_circle_area(radius=2) == 4 * pi, 'get_circle_area(radius=2) (3 pts visible)'",
            "output": "Error before assert statement run",
            "score": 0.0,
            "status": "failed",
            "visibility": "visible"
        },
        {
            "max_score": 2.0,
            "name": "assert get_circle_area(radius=10) == 100 * pi, 'get_circle_area(radius=10) (2 pts hidden)'",
            "output": "Error before assert statement run",
            "score": 0.0,
            "status": "failed",
            "visibility": "hidden"
        }
    ]
}